Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/RemoteMongoClientIntTests.swift

```diff
@@ -26,6 +27,17 @@ class RemoteMongoClientIntTests: BaseStitchIntTestCocoaTouch {
         try! prepareService()
     }
 
+    override func tearDown() {
+        let joiner = CallbackJoiner()
+        getTestColl().deleteMany([:], joiner.capture())
```

Comment:
[nice] This is soooo much nicer than XCTestExpectation, I wish we had this before